apps/profiles/viewsets.py

```diff
@@ -0,0 +1,42 @@
+
+from rest_framework import viewsets
+
+from rest_framework.permissions import IsAuthenticatedOrReadOnly
+
+from apps.profiles.serializers import (
+    UserRatingListSerializer,
+    UserRatingUpdateSerializer,
+    UserRatingCreateSerializer,
+    UserRatingRetrieveSerializer,
+)
+from apps.profiles.permissions import UserRatingPermissions
+from apps.profiles.models import UserRating
+
+
+class UserRatingViewSet (viewsets.ModelViewSet):
+    """
+    A simple ViewSet for viewing and editing user's ratings.
+    """
+    model = UserRating
+    queryset = UserRating.objects.all()
+    lookup_url_kwarg = 'rating_id'
+    permission_classes = [IsAuthenticatedOrReadOnly, UserRatingPermissions]
+    http_method_names = ["post", "get", "put", "delete"]
+
+    def get_queryset(self):
+        if self.kwargs.get("rating_id"):
+            self.queryset = UserRating.objects.filter(user_rated_id=self.kwargs["user_id"], id=self.kwargs["rating_id"])
```

Comment:
By doing it this way you get a more complicated code, because for example if you need to change queryset for two cases you need to edit two strings.
that's why it's better to separate
`queryset =  UserRating.object
`queryset = `queryset.filter(....)`